docs/topics/wasm/wasm-debugging.md

```diff
@@ -0,0 +1,149 @@
+[//]: # (title: Debug Kotlin/Wasm code)
+
+> Kotlin/Wasm is in [Alpha](components-stability.md). It may be changed at any time.
+>
+{type="note"}
+
+This tutorial demonstrates how to debug in the browser a [Compose Multiplatform](https://www.jetbrains.com/lp/compose-multiplatform/)
+application built with Kotlin/Wasm.
+
+## Before you start
+
+Create a project using the Kotlin Multiplatform wizard:
+
+1. Open the [Kotlin Multiplatform wizard](https://kmp.jetbrains.com/#newProject).
+2. On the **New Project** tab, you can change the project name to "WasmDemo" and the project ID to "wasm.project.demo".
+3. Select the **Web** option.
+4. Click the **Download** button and unpack the resulting archive.
+
+![Kotlin Multiplatform wizard](wasm-compose-wizard.png){width=600}
+
+## Open the project in IntelliJ IDEA
+
+1. Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/).
+2. On the Welcome screen of IntelliJ IDEA, click **Open** or select **File | Open** in the menu bar.
+3. Navigate to the unpacked "WasmDemo" folder and click **Open**.
+
+## Run the application
+
+1. In IntelliJ IDEA, open the **Gradle** tool window by selecting **View** | **Tool Windows** | **Gradle**.
+
+   > You need at least Java 11 as your Gradle JVM for the tasks to load successfully.
+   >
+   {type="note"}
+
+2. In **composeApp** | **Tasks** | **kotlin browser**, select and run the **wasmJsBrowserRun** task.
+
+   ![Run the Gradle task](wasm-gradle-task-window.png){width=600}
+
+   Alternatively, you can run the following command in the terminal from the `WasmDemo` root directory:
+
+   ```bash
+   ./gradlew wasmJsBrowserRun
+   ```
+
+3. Once the application starts, open the following URL in your browser:
+
+   ```bash
+   http://localhost:8080/
+   ```
+
+   >The port number can vary because the 8080 port may be unavailable. You can find the actual port number printed
+   > in the Gradle build console.
+   >
+   {type="tip"}
+
+   You can see a "Click me!" button. Click it:
```

Comment:
Some weeks ago I received feedback from our colleague Aleksey to change this sentence so that it gives more transition to the expected result. I got his point and it made sense to me to change it. His suggestion was also "should". How does it sound to you with "should"?

Reply:
Interesting! For me, the additional "can" is unnecessary and as a native speaker it somehow infers that it was obscured before in some way. I like to follow [Google's guide](https://developers.google.com/style/word-list#can) in this instance. "Should" is conditional so I would avoid that, for sure! Otherwise it makes it sound like we aren't 100% sure what will happen. I leave the final decision up to you about keeping or removing the "can". 😄 I don't have strong feelings about it.